jans-cedarling/bindings/cedarling_ruby/ext/cedarling/build.rs

```diff
@@ -0,0 +1,153 @@
+/*
+ * This software is available under the Apache-2.0 license.
+ * See https://www.apache.org/licenses/LICENSE-2.0.txt for full text.
+ *
+ * Copyright (c) 2024, Gluu, Inc.
+ */
+
+fn manifest_dir() -> std::path::PathBuf {
+    let manifest_dir = std::env::var("CARGO_MANIFEST_DIR").expect("CARGO_MANIFEST_DIR not in build script. Wut!?");
+    std::path::Path::new(&manifest_dir).to_path_buf()
+}
+
+// Save the current core cedarling version in an ENV var for use by the rust
+// side of the ruby binding.
+fn cedarling_version_in_env() {
+    // Fetch cedarling version from its Cargo.toml
+
+    let cedarling_cargo_toml : &str = "../../../../cedarling/Cargo.toml";
```

Comment:
In this case I think no.

Because `build.rs` is for a package of the workspace located at the relative path. So it needs to be really specific, and should not be a variable.